### PR TITLE
Document converting to plain Date

### PR DESCRIPTION
**Pull request by oxinabox**
document #222

### CODE REVIEW

File: docs/src/conversions.md [hunk 1]
```diff
@@ -6,6 +6,42 @@ DocTestSetup = quote
 end
 ```
 
+## Converting to plain Dates types without timezone information.
+
+To convert from a `ZonedDateTime` into a vanillia `DateTime`, one can use the `DateTime` constructor.
```
Comment (nickrobinson251) [inline docs/src/conversions.md]:
```suggestion
To convert from a `ZonedDateTime` into a vanilla `DateTime`, one can use the `DateTime` constructor.
```

File: docs/src/conversions.md [hunk 2]
```diff
@@ -6,6 +6,42 @@ DocTestSetup = quote
...
+Passing either `Local` to directly drop the timezone,  or `UTC` to convert to UTC time first.
+The canonical way to represent datetimes is generally in `UTC`, this is a requirement to correctly compute the [Unix Timestamp](https://en.wikipedia.org/wiki/Unix_time).
```
Comment (nickrobinson251) [inline docs/src/conversions.md]:
```suggestion
The canonical way to represent datetimes is generally in `UTC`, as this is a requirement to correctly compute the [Unix Timestamp](https://en.wikipedia.org/wiki/Unix_time).
```

File: docs/src/conversions.md [hunk 3]
```diff
@@ -6,6 +6,42 @@ DocTestSetup = quote
...
+
+```jldoctest
+julia> x = ZonedDateTime(DateTime(2020, 11, 30, 22, 35), tz"America/Winnipeg")
```
Comment (nickrobinson251) [inline docs/src/conversions.md]:
```suggestion
julia> x = ZonedDateTime(2020, 11, 30, 22, 35, tz"America/Winnipeg")
```

File: docs/src/conversions.md [hunk 4]
```diff
@@ -6,6 +6,42 @@ DocTestSetup = quote
...
+2020-11-30T22:35:00
+
+julia> DateTime(x, UTC)
+2020-12-01T04:35:00
+```
+
+Similar can be done for `Date` and `Time`:
+
+```jldoctest
+julia> x = ZonedDateTime(DateTime(2020, 11, 30, 22, 35), tz"America/Winnipeg")
```
Comment (nickrobinson251) [inline docs/src/conversions.md]:
```suggestion
julia> x = ZonedDateTime(2020, 11, 30, 22, 35, tz"America/Winnipeg")
```

**Comment by oxinabox** [comment]
Failing on julia nightly only
https://github.com/JuliaTime/TimeZones.jl/issues/246

File: docs/src/conversions.md [hunk 5]
```diff
@@ -6,6 +6,42 @@ DocTestSetup = quote
...
+```jldoctest
+julia> x = ZonedDateTime(2020, 11, 30, 22, 35, tz"America/Winnipeg")
+2020-11-30T22:35:00-06:00
+
+julia> DateTime(x, Local)
+2020-11-30T22:35:00
+
+julia> DateTime(x, UTC)
+2020-12-01T04:35:00
+```
```
Comment (omus) [inline docs/src/conversions.md]: Can/should we use the same example as the docstring?

File: docs/src/conversions.md [hunk 6]
```diff
@@ -6,6 +6,42 @@ DocTestSetup = quote
 end
 ```
 
+## Converting to plain Dates types without timezone information.
+
+To convert from a `ZonedDateTime` into a vanilla `DateTime`, one can use the `DateTime` constructor.
+Passing either `Local` to directly drop the timezone,  or `UTC` to convert to UTC time first.
```
Comment (omus) [inline docs/src/conversions.md]: I'm hesitant to use the word "convert" as `UTC` is just returning a stored value where `Local` is where the conversion occurs:
```suggestion
Passing either `Local` to directly drop the time zone,  or `UTC` to extract the time as UTC instead.
```

File: docs/src/conversions.md [hunk 7]
```diff
@@ -6,6 +6,42 @@ DocTestSetup = quote
 end
 ```
 
+## Converting to plain Dates types without timezone information.
```
Comment (omus) [inline docs/src/conversions.md]:
```suggestion
## Converting Date/Time without time zone information
```

**Pull request merged**